src/aspire/basis/fb_3d.py

```diff
@@ -146,6 +146,8 @@ def evaluate(self, v):
             This is an array whose first dimensions equal `self.z` and the
             remaining dimensions correspond to dimensions two and higher of `v`.
         """
+
+        v = v.T
```

Comment:
So the FB3D basis had the wrong ordering, am I understanding correctly?

Reply:
Yes, at the time I focused more on updating FFB3D; I incorrectly thought that was what we used in ASPIRE. I narrowed down to FB being different from FFB leading to trouble using the estimator. That should work with either basis class interchangeably. I always had something wrong here...  I moved the supporting code (mdim_fun.. and estimator) toward row major and just do this transpose in and out of the reference FBBasis evaluate/evaluate_t so I could otherwise leave that code alone. (Also the corresponding utest).